# --- ROCK-COPYRIGHT-NOTE-BEGIN ---
# 
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# Please add additional copyright information _after_ the line containing
# the ROCK-COPYRIGHT-NOTE-END tag. Otherwise it might get removed by
# the ./scripts/Create-CopyPatch script. Do not edit this copyright text!
# 
# ROCK Linux: rock-src/package/mnemoc/vserver/no_extraversion.diff
# ROCK Linux is Copyright (C) 1998 - 2003 Clifford Wolf
# 
# This program is free software; you can redistribute it and/or modify
# it under the terms of the GNU General Public License as published by
# the Free Software Foundation; either version 2 of the License, or
# (at your option) any later version. A copy of the GNU General Public
# License can be found at Documentation/COPYING.
# 
# Many people helped and are helping developing ROCK Linux. Please
# have a look at http://www.rocklinux.org/ and the Documentation/TEAM
# file for details.
# 
# --- ROCK-COPYRIGHT-NOTE-END ---

--- ./ctx-17.patch.orig	2003-07-10 00:43:23.000000000 -0400
+++ ./ctx-17.patch	2003-07-10 00:43:51.000000000 -0400
@@ -1,15 +1,3 @@
-diff -NurbP --minimal linux-2.4.21/Makefile linux-2.4.21-ctx17/Makefile
---- linux-2.4.21/Makefile	Fri Jun 13 17:49:08 2003
-+++ linux-2.4.21-ctx17/Makefile	Fri Jun 13 17:54:29 2003
-@@ -1,7 +1,7 @@
- VERSION = 2
- PATCHLEVEL = 4
- SUBLEVEL = 21
--EXTRAVERSION =
-+EXTRAVERSION = -ctx17
- 
- KERNELRELEASE=$(VERSION).$(PATCHLEVEL).$(SUBLEVEL)$(EXTRAVERSION)
- 
 diff -NurbP --minimal linux-2.4.21/arch/i386/kernel/entry.S linux-2.4.21-ctx17/arch/i386/kernel/entry.S
 --- linux-2.4.21/arch/i386/kernel/entry.S	Fri Jun 13 17:49:08 2003
 +++ linux-2.4.21-ctx17/arch/i386/kernel/entry.S	Fri Jun 13 17:54:29 2003
